lib/game.rb

```diff
@@ -36,31 +38,45 @@ def initialize(
     def play
         while status == :play
             output_board
-            position = get_valid_move(board: game_state.board, presenter: game_io)
-            board = update_board(input: position)
-            update_game_status(board: board)
-            game_state.switch_players if status == :play
+            output_save_option
+            input = get_valid_input(board: game_state.board, presenter: game_io)
+            if input == InputType::SAVE
+                @status = :save
+            else
+                board = update_board(input: input) 
+                update_game_status(board: board)
+                game_state.switch_players if status == :play
+            end
         end
         output_board
+        save_game if status == :save
         output_win if status == :win
         output_tie if status == :tie 
     end
 
     private
-    attr_accessor :game_io, :board_presenter
+    attr_accessor :game_io
 
     def output_board
         game_io.clear
         board_presenter.generate(board: game_state.board)
         game_io.output_message(board_presenter.output)
     end
 
-    def get_valid_move(board:, presenter:)
+    def output_save_option
+        game_io.output_message(Messages.output_save_option)
+    end
+
+    def get_valid_input(board:, presenter:)
```

Comment:
Bit of a nitpick, but it is typically conventional to avoid prefixing method names with `get` or `set` for accessors and mutators. (Reference: https://github.com/rubocop-hq/ruby-style-guide#accessor_mutator_method_names)